controllers/team.controller.js

```diff
@@ -0,0 +1,61 @@
+const Team = require('../models').Team;
+
+const defaultResponse= require('../utils/defaultResponse');
+const constants= require('../utils/constants');
+const responseStatus= require('../utils/responseStatus');
+
+exports.getTeam = async (req, res) => {
+    try {
+        const team = await Team.findAll();
+        if (team) {
```

Comment:
checking for object validation should always be 
`if (!team) return res.status(404).message('team with id not found')` or similar